group_vars/matrix_servers

```diff
@@ -986,7 +986,7 @@ matrix_bot_honoroit_systemd_required_services_list: |
 # Postgres is the default, except if not using `matrix_postgres` (internal postgres)
 matrix_bot_honoroit_database_engine: "{{ 'postgres' if matrix_postgres_enabled else 'sqlite' }}"
 matrix_bot_honoroit_database_password: "{{ '%s' | format(matrix_homeserver_generic_secret_key) | password_hash('sha512', 'honoroit.bot.db') | to_uuid }}"
-matrix_bot_honoroit_container_image_self_build: "{{ matrix_architecture != 'amd64' }}"
+matrix_bot_honoroit_container_image_self_build: false
```

Comment:
Probably best to change this to:

```yaml
matrix_bot_honoroit_container_image_self_build: "{{ matrix_architecture not in ['amd64', 'arm32', 'arm64'] }}"
```

Reasons:

- in case some other architecture comes up in the future (well, unlikely, but..), self-building will kick in. Well, it will probably fail as described in #1518, but..
- doing that (instead of hardcoding it to `false`) seems more consistent with what we do for other components in `group_vars/matrix_servers`.

Alternatively, if we'd be setting this to `false`, we may as well not define it in `group_vars/matrix_servers` at all. The default value is `false` anyway (coming from the role's `defaults/main.yml` file).